Source/Krypton Toolkit/Directory.Build.targets

```diff
@@ -132,9 +132,9 @@
 
 
 				<Description Condition="'$(TFMs)' == 'all'">
-					An extension to the Standard Toolkit, which supports .NET Framework 4.6.2 - 4.8, .NET Core 3.1 and .NET 5/6. $(Description)
+					An extension to the Standard Toolkit, which supports .NET Framework 4.6.2 - 4.8 and .NET 5/6. $(Description)
```

Comment:
Should also remove .net 5

Reply:
And include 4.8.1

Reply:
How did I miss that? Anyway, fixed now :)